### PR TITLE
Implement circle

### PR DESCRIPTION
**Pull request by Panz99**

### CODE REVIEW

**Comment by lukas-h** [comment]
Hello @Panz99, thanks for this contribution! To merge it, we need some unit tests. Most likely you can find some in the turf js repository

**Comment by Panz99** [comment]
@lukas-h The unit test for the circle function was added

File: lib/circle.dart [hunk 1]
```diff
@@ -0,0 +1,4 @@
+library turf_transform;
```
Comment (lukas-h) [inline lib/circle.dart:1]: needs to be renamed to `library turf_circle;`

**Review by lukas-h — APPROVED** [review]

**Comment by lukas-h** [comment]
@Panz99  Thank you very much for your contribution. I just merged it. We will publish your contribution in the next minor release.

**Pull request merged**